### ISSUE TITLE
optional dependency reportlab is not optional

### ISSUE THREAD

**Issue by wagner-certat** [MISP/PyMISP#550]
reportlab is an optional dependency of the `reportlab_generator`:

https://github.com/MISP/PyMISP/blob/eff7146b3c4a981c9e13fa9a4c403950dfddd503/pymisp/tools/reportlab_generator.py#L22-L37

But the imported Flowable is then a hard requirement:

https://github.com/MISP/PyMISP/blob/eff7146b3c4a981c9e13fa9a4c403950dfddd503/pymisp/tools/reportlab_generator.py#L52

**Comment by Rafiot** [comment]
Yeah, this file is definitely not supposed to be imported unless you have that dep. In which context does it trigger an exception? This file isn't imported in __init__.py.

**Comment by wagner-certat** [comment]
> Yeah, this file is definitely not supposed to be imported unless you have that dep. In which context does it trigger an exception? This file isn't imported in `__init__.py`

Testing - that loads every file to look for tests.

**Comment by Rafiot** [comment]
Oh yeah, sure, testing does, that was a shortcut. Well, I can add reportlab in the requirement for testing.

That module is only supposed to be used if you know what youŕe doing.
